web/src/components/JSMpegPlayer.jsx

```diff
@@ -8,6 +8,20 @@ export default function JSMpegPlayer({ camera, width, height }) {
   const url = `${baseUrl.replace(/^http/, 'ws')}/live/${camera}`
 
   useEffect(() => {
+    let wakeLock = null;
+
+    const requestWakeLock = async () => {
+      if (!navigator.wakeLock) {
+        return;
+      }
+
+      try {
+        wakeLock = await navigator.wakeLock.request('screen');
```

Comment:
Does Safari have a polyfill?

Reply:
:( Seems like the only way to use in iOS would be to either play a hidden video on a loop or to do some fake browser navigations using anchors: https://github.com/richtr/NoSleep.js/blob/master/src/index.js

Reply:
What about using an html5 video element? This seems like a more natural way to support fullscreen but I am not sure what issues there might be with it.